### ISSUE TITLE
Template polymer-init-starter-kit:app not found

### ISSUE THREAD

**Issue by ernsheong** [PolymerElements/polymer-starter-kit#1039]
### Description
Run `polymer init starter-kit`

### Expected outcome
Init works successfully

### Actual outcome
```
info:    Running template polymer-init-starter-kit:app...
error:   Template polymer-init-starter-kit:app not found
error:   cli runtime exception: Error: Template polymer-init-starter-kit:app not found
error:   Error: Template polymer-init-starter-kit:app not found
    at Object.<anonymous> (/Users/ernsheong/.nvm/versions/node/v6.9.5/lib/node_modules/polymer-cli/lib/init/init.js:220:19)
    at next (native)
    at fulfilled (/Users/ernsheong/.nvm/versions/node/v6.9.5/lib/node_modules/polymer-cli/lib/init/init.js:17:58)
    at process._tickCallback (internal/process/next_tick.js:103:7)
    at Module.runMain (module.js:606:11)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3

```

### Steps to reproduce
1. `npm i -g polymer-cli`
2. `npm i -g bower`
3. `polymer init starter-kit`

### Browsers Affected

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE 11
- [ ] Safari 8
- [ ] Safari 9

polymer-cli version: 1.2.0

**Comment by ernsheong** [comment]
Looks like README needs an update from 

```
polymer init starter-kit
```

to just 

```
polymer init
```
?

**Comment by moderndeveloperllc** [comment]
@ernsheong This is probably a question for the polymer-cli repository, but [looking through that code](https://github.com/Polymer/polymer-cli/blob/master/src/init/init.ts), you might want to try `polymer init polymer-2-starter-kit` if you want a PSK3/Polymer 2 application.

**Comment by refactormyself** [comment]
@moderndeveloperllc `polymer init polymer-2-starter-kit` works fine but I still think @ernsheong point is valid, the [ReadMe](https://github.com/PolymerElements/polymer-starter-kit#initialize-project-from-template) needs to be edited it says `polymer init starter-kit`

**Comment by ernsheong** [comment]
New PR for that: https://github.com/PolymerElements/polymer-starter-kit/pull/1042

**Comment by canarslan12** [comment]
> ReadMe needs to be edited it says polymer init starter-kit

+1

**Comment by abdonrd** [comment]
Closed by #1042.
